### ISSUE TITLE
Expect Units of Measure names to be camel case

### ISSUE THREAD

**Issue by jbeeko** [fsprojects/FSharpLint#425]
### Description

Unit of Measure names are expected to be PascalCase by default. I believe this is a regression of this issue #139

### Repro steps

Create a file called `test.fsx` with the following content:
```
[<Measure>] type kg
```
Install FSharpLint and run:
`$ dotnet fsharplint lint test.fsx`
### Expected behavior

No lint warnings.

### Actual behaviour

The following warning is raised.
```
$ dotnet fsharplint lint test.fsx
========== Linting test.fsx ==========
Consider changing `kg` to PascalCase.
Error on line 1 starting at column 17
[<Measure>] type kg
                 ^ 
--------------------------------------------------------------------------------
========== Finished: 1 warnings ==========
========== Summary: 1 warnings ==========
$ 
```

### Known workarounds

Create a configuration file that changes the configuration for the rule. 

### Related information

* MacOS
* FSharpLint installed globally on May 18, 2020
* .NET 5.0 preview 3

**Comment by jgardella** [comment]
Thanks @jbeeko for the detailed report, this was fixed in #427 and released in v0.14.1.
